<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>color_preprocess_config_screen</class>
 <widget class="QWidget" name="color_preprocess_config_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#E5E5E5</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="8,2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="containerScreen" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="screen1">
        <property name="styleSheet">
         <string notr="true">background-color: #AFF;font-weight:bold;</string>
        </property>
        <property name="text">
         <string>SCREEN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="screen2">
        <property name="styleSheet">
         <string notr="true">background-color: #AFF;font-weight:bold;</string>
        </property>
        <property name="text">
         <string>SCREEN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="containerConfig" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #EEEEEE</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align:center;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>COLOR COMPARISION - PREPROCESS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="containerParam" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6,2">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="containerLeft" native="true">
           <property name="styleSheet">
            <string notr="true">#containerLeft {
	border: 1px solid #A5A5A5
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,0">
            <item alignment="Qt::AlignTop">
             <widget class="QCheckBox" name="chkColorCompare">
              <property name="text">
               <string>Enable Color Comparison</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="containerMid" native="true">
           <property name="styleSheet">
            <string notr="true">#containerMid {
	border: 1px solid #A5A5A5
}</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupCbbResize">
              <property name="title">
               <string>Resize (Width - Height)</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QComboBox" name="cbbResizeWidth">
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">height:22px</string>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>Item 1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Item 2</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cbbResizeHeight">
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">height:22px</string>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>Item 1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Item 2</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupSldBrightLeft">
              <property name="title">
               <string>Brightness left: 0</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QSlider" name="sldBrightLeft">
                 <property name="maximum">
                  <number>30</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QGroupBox" name="groupSldConstrastLeft">
              <property name="title">
               <string>Contrast left: -200</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QSlider" name="sldConstrastLeft">
                 <property name="minimum">
                  <number>-40</number>
                 </property>
                 <property name="maximum">
                  <number>40</number>
                 </property>
                 <property name="value">
                  <number>-40</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QGroupBox" name="groupSldBrightRight">
              <property name="title">
               <string>Brightness right: 0</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QSlider" name="sldBrightRight">
                 <property name="maximum">
                  <number>30</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupSldBlur">
              <property name="title">
               <string>Blur: 0</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QSlider" name="sldBlur">
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupSldSaturation">
              <property name="title">
               <string>Saturation: 0</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QSlider" name="sldSaturation">
                 <property name="maximum">
                  <number>10</number>
                 </property>
                 <property name="pageStep">
                  <number>2</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QGroupBox" name="groupSldConstrastRight">
              <property name="title">
               <string>Contrast right: -200</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QSlider" name="sldConstrastRight">
                 <property name="minimum">
                  <number>-40</number>
                 </property>
                 <property name="maximum">
                  <number>40</number>
                 </property>
                 <property name="value">
                  <number>-40</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="containerRight" native="true">
           <property name="styleSheet">
            <string notr="true">#containerRight {
	border: 1px solid #A5A5A5
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item alignment="Qt::AlignTop">
             <widget class="QWidget" name="containerVerticalBtn" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QWidget" name="containerNavBtn" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnBack">
                 <property name="text">
                  <string>BACK</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnNext">
                 <property name="text">
                  <string>NEXT</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
